src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -80,6 +80,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.2.0" />
+    <PackageReference Include="System.DirectoryServices" Version="4.0.0-preview3-25423-02" />
```

Comment:
This doesn't appear to be a supported package, perhaps we should not support AD rights for now.

Reply:
Official package is  [System.DirectoryService](https://dotnet.myget.org/feed/dotnet-core/package/nuget/System.DirectoryServices). It is 4.5.0 version. @daxian-dbw said that we should review the API before use.